js/views/composer.js

```diff
@@ -187,6 +208,22 @@ define(function(require) {
 				_this.saveDraft();
 			}, this.draftInterval);
 		},
+		toggleEditor: function()
```

Comment:
>we couldn't see that button on the interface.

may be a problem with cached js. did you tried opening in incognito mode to test cached js